remote-messaging/remote-messaging-api/src/main/java/com/duckduckgo/remote/messaging/api/AttributeMatcherPlugin.kt

```diff
@@ -17,9 +17,19 @@
 package com.duckduckgo.remote.messaging.api
 
 interface AttributeMatcherPlugin {
-    suspend fun evaluate(matchingAttribute: MatchingAttribute<*>): Boolean?
+    suspend fun evaluate(matchingAttribute: MatchingAttribute): Boolean?
 }
 
-interface MatchingAttribute<T> {
-    fun matches(matchingValue: T): Boolean?
+interface MatchingAttribute
```

Comment:
I've removed the method `matches` from the interface and the generic from the interface, they are not adding any value, and it's simpler like this.